ssg/build_yaml.py

```diff
@@ -21,7 +21,7 @@ def add_sub_element(parent, tag, data):
     # TODO: Remove this function after we move to Markdown everywhere in SSG
     try:
         ustr = str("<{0}>{1}</{0}>").format(tag, data)
-    except NameError:
+    except UnicodeEncodeError:
```

Comment:
we discussed this and we need a different solution here. consider this:
we are using python3, the unicode encode causes an error because the utf-8 is malformed. this will get us to the line under this comment where it will raise a NameError because unicode() is not defined on python3. This hides the exception and doesn't tell you where the error is.